### PR TITLE
Refactor background js

### PR DESCRIPTION
**Pull request by ltphan**
Converted a Promise to async/await for better readability.

### CODE REVIEW

File: fetch-crypto-assets.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const file = "./data.json"
+
+async function getJSON(jsonFile) {
```
Comment (kenny-nguyen-mgf) [inline fetch-crypto-assets.js:3]: What do you think about renaming the param to simply `url`? `jsonFile` kinda implies that you're providing a json file as a paramter (not true)

File: fetch-crypto-assets.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+}
+
+function buildURL(param) {
+    return `https://api.coinbase.com/v2/prices/${param}/spot`
+}
+
+async function getURLS() {
+    try {
+        const params = await getJSON(file)
```
Comment (kenny-nguyen-mgf) [inline fetch-crypto-assets.js:19]: does `const params = file` not work? 🤔

File: fetch-crypto-assets.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        const urls = params.map((val) => {
```
Comment (kenny-nguyen-mgf) [inline fetch-crypto-assets.js]: you can simply return immediately

```js
return params.map(...)

File: fetch-crypto-assets.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            promises.push(fetchData(urls[i]))
+        }
+
+        return promises
+    } catch (e) {
+        throw new Error("Something went wrong building list of promises", e)
+    }
+}
+
+async function fetchMultipleCalls() {
```
Comment (kenny-nguyen-mgf) [inline fetch-crypto-assets.js]: 🥳

File: fetch-crypto-assets.js (hunk 2 above)
Comment (ltphan) [reply fetch-crypto-assets.js:19]: After our conversation, cannot use `module.exports` in a non-Node environment. This is just pure vanilla JS.

File: fetch-crypto-assets.js (hunk 3 above)
Comment (ltphan) [reply fetch-crypto-assets.js]: Addressed: fc516e8a0fc15b17133c902a76d79e362313af8e

**Comment by ltphan** [comment]
Screen Shot 2021-06-30 at 11 41 35 AM

**Pull request merged**